.github/workflows/build.yml

```diff
@@ -71,47 +72,55 @@ jobs:
           - name: linux 3.6 amd64
             os: ubuntu-latest
             pyver: cp36-cp36m
-            piparch: manylinux1_x86_64
+            piparch: manylinux_2_28_x86_64
```

Comment:
Be careful when changing `piparch` to avoid potentially subtle ABI breakage (eg. https://github.com/epics-base/p4p/issues/109#issuecomment-1597670767).  It would be best if the whole chain of epicscorelibs, pvxslibs, and p4p upload wheels built for the same manylinux image.

This is one of the annoying places where it is necessary to coordinate a change to the whole P4P "stack".  If updating the py < 3.12 isn't immediately necessary, I would suggest deferring to another set of PRs.

That said, at this point I'm ok with moving past manylinux1 except for py2.7.

Reply:
I have reverted the piparch changes and will move them into a new PR